### ISSUE TITLE
Download code bindings for Typescript 3+ generates code with a typo in marshaller.ts file

### ISSUE THREAD

**Issue by DavidAbram** [aws/aws-toolkit-vscode#1160]
**Describe the bug**

When downloading code bindings, in the auto-generated file "marshaller.ts" (./marshaller/marshaller.ts) there is an error in a import statement; ```import { AWSEvent } from '../aWSEvent';``` should be ```import { AWSEvent } from '../AWSEvent';```

**To Reproduce**

In vs-code is selected the AWS tab I selected discovered-schemas, selected download Code Binding on the following schema:

`{
  "openapi": "3.0.0",
  "info": {
    "version": "1.0.0",
    "title": "Todo"
  },
  "paths": {},
  "components": {
    "schemas": {
      "AWSEvent": {
        "type": "object",
        "required": [
          "detail-type",
          "resources",
          "detail",
          "id",
          "source",
          "time",
          "region",
          "version",
          "account"
        ],
        "x-amazon-events-detail-type": "todo",
        "x-amazon-events-source": "example.api.todo",
        "properties": {
          "detail": {
            "$ref": "#/components/schemas/Todo"
          },
          "account": {
            "type": "string"
          },
          "detail-type": {
            "type": "string"
          },
          "id": {
            "type": "string"
          },
          "region": {
            "type": "string"
          },
          "resources": {
            "type": "array",
            "items": {
              "type": "object"
            }
          },
          "source": {
            "type": "string"
          },
          "time": {
            "type": "string",
            "format": "date-time"
          },
          "version": {
            "type": "string"
          }
        }
      },
      "Todo": {
        "type": "object",
        "required": [
          "eventID",
          "awsRegion",
          "eventSourceARN",
          "eventVersion",
          "eventSource",
          "eventName",
          "dynamodb"
        ],
        "properties": {
          "dynamodb": {
            "$ref": "#/components/schemas/Dynamodb"
          },
          "awsRegion": {
            "type": "string"
          },
          "eventID": {
            "type": "string"
          },
          "eventName": {
            "type": "string"
          },
          "eventSource": {
            "type": "string"
          },
          "eventSourceARN": {
            "type": "string"
          },
          "eventVersion": {
            "type": "string"
          }
        }
      },
      "Dynamodb": {
        "type": "object",
        "required": [
          "NewImage",
          "ApproximateCreationDateTime",
          "Keys"
        ],
        "properties": {
          "Keys": {
            "$ref": "#/components/schemas/Keys"
          },
          "NewImage": {
            "$ref": "#/components/schemas/NewImage"
          },
          "ApproximateCreationDateTime": {
            "type": "number"
          }
        }
      },
      "Keys": {
        "type": "object",
        "required": [
          "id"
        ],
        "properties": {
          "id": {
            "type": "string"
          }
        }
      },
      "NewImage": {
        "type": "object",
        "required": [
          "createdAt",
          "abc",
          "color",
          "size",
          "checked",
          "isBig",
          "stringAndNumber",
          "id",
          "text",
          "stringOrNumber",
          "updatedAt"
        ],
        "properties": {
          "abc": {
            "$ref": "#/components/schemas/Abc"
          },
          "checked": {
            "type": "boolean"
          },
          "color": {
            "type": "number"
          },
          "createdAt": {
            "type": "integer",
            "format": "int64"
          },
          "id": {
            "type": "string"
          },
          "isBig": {
            "type": "boolean"
          },
          "size": {
            "type": "number"
          },
          "stringAndNumber": {
            "type": "array",
            "items": {
              "oneOf": [
                {
                  "type": "string"
                },
                {
                  "type": "number"
                }
              ]
            }
          },
          "stringOrNumber": {
            "type": "number"
          },
          "text": {
            "type": "string"
          },
          "updatedAt": {
            "type": "integer",
            "format": "int64"
          }
        }
      },
      "Abc": {
        "type": "object"
      }
    }
  }
}`

The generated code (./marshaller/marshaller.ts) had a typo in a import statment.

**Expected behavior**

```import { AWSEvent } from '../aWSEvent';``` should be ```import { AWSEvent } from '../AWSEvent';```

**Desktop:**

- OS:  Linux x64 5.4.0-7634-generic
- Visual Studio Code Version:  1.46.0
- AWS Toolkit Version:  1.10.0

**Comment by ejafarli** [comment]
Thanks for reporting this bug. We have a fix that is pending deployment to all regions. I'll update the ticket once the deployment is completed.

**Comment by leantorres73** [comment]
That's right, the marshaller is setting wrong name import (`AWSEvent`). Any progress on this?

**Comment by ejafarli** [comment]
@leantorres73 Did you reproduce it by downloading one of the schemas under `aws.events` registry? I will give an update once we have a finalized date for deployments.

**Comment by leantorres73** [comment]
@ejafarli the issue is not with the AWS events registry because their naming is AWSEvent always. The problem is with custom registry, it is still downloading the event as `AWSEvent` and that's wrong

**Comment by ejafarli** [comment]
@leantorres73 if you download typescript code bindings for one of the schemas under `aws.events`, then there is a lower case import issue `import { AWSEvent } from '../aWSEvent'` as described in this ticket.  Is your issue related to the this specific ticket, if yes can you elaborate on how to reproduce?

**Comment by leantorres73** [comment]
@ejafarli  you're right, in my case I don't get that typo issue.

It's a new issue that the import is always a fixed value. So for custom schemas this doesn't work.
for example if I create a schema with the name `NEWTESTSCHEMA` the import it's still

`import { AWSEvent } from '../AWSEvent';`


```
{ "openapi": "3.0.0", "info": { "version": "1.0.0", "title": "Todo" }, "paths": {}, "components": { "schemas": { "NEWTESTSCHEMA": { "type": "object", "required": [ "detail-type", "resources", "detail", "id", "source", "time", "region", "version", "account" ], "x-amazon-events-detail-type": "todo", "x-amazon-events-source": "example.api.todo", "properties": { "detail": { "$ref": "#/components/schemas/Todo" }, "account": { "type": "string" }, "detail-type": { "type": "string" }, "id": { "type": "string" }, "region": { "type": "string" }, "resources": { "type": "array", "items": { "type": "object" } }, "source": { "type": "string" }, "time": { "type": "string", "format": "date-time" }, "version": { "type": "string" } } }, "Todo": { "type": "object", "required": [ "eventID", "awsRegion", "eventSourceARN", "eventVersion", "eventSource", "eventName", "dynamodb" ], "properties": { "dynamodb": { "$ref": "#/components/schemas/Dynamodb" }, "awsRegion": { "type": "string" }, "eventID": { "type": "string" }, "eventName": { "type": "string" }, "eventSource": { "type": "string" }, "eventSourceARN": { "type": "string" }, "eventVersion": { "type": "string" } } }, "Dynamodb": { "type": "object", "required": [ "NewImage", "ApproximateCreationDateTime", "Keys" ], "properties": { "Keys": { "$ref": "#/components/schemas/Keys" }, "NewImage": { "$ref": "#/components/schemas/NewImage" }, "ApproximateCreationDateTime": { "type": "number" } } }, "Keys": { "type": "object", "required": [ "id" ], "properties": { "id": { "type": "string" } } }, "NewImage": { "type": "object", "required": [ "createdAt", "abc", "color", "size", "checked", "isBig", "stringAndNumber", "id", "text", "stringOrNumber", "updatedAt" ], "properties": { "abc": { "$ref": "#/components/schemas/Abc" }, "checked": { "type": "boolean" }, "color": { "type": "number" }, "createdAt": { "type": "integer", "format": "int64" }, "id": { "type": "string" }, "isBig": { "type": "boolean" }, "size": { "type": "number" }, "stringAndNumber": { "type": "array", "items": { "oneOf": [ { "type": "string" }, { "type": "number" } ] } }, "stringOrNumber": { "type": "number" }, "text": { "type": "string" }, "updatedAt": { "type": "integer", "format": "int64" } } }, "Abc": { "type": "object" } } } }
```


I should open a new ticket for this!

**Comment by ejafarli** [comment]
@leantorres73 Yes, in that case please open a new ticket if you haven't already
